### PR TITLE
Default .editorconfig file

### PR DESCRIPTION
**Pull request by tallnato**

### CODE REVIEW

File: Backend/templates/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+max_line_length = 120
+
+
+[*.{yaml,yml}]
+indent_size = 2
+
+
+[*.go]
+indent_style = space
+indent_size = 4
```
Comment (hugomsilvam) [inline Backend/templates/.editorconfig]: you have already defined `indent_size = 4`, for all the files. Could you delete this? :thinking:

**Review by hugomsilvam — COMMENTED** [review]
looks good :cake:

File: Backend/templates/.editorconfig (hunk 1 above)
Comment (tallnato) [reply Backend/templates/.editorconfig]: nice catch 👍

**Review by ruimartinsptl — APPROVED** [review]

File: Backend/templates/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+insert_final_newline = true
+max_line_length = 120
+
+
+[*.{yaml,yml}]
+indent_size = 2
+
+
+[*.go]
+indent_style = space
```
Comment (portellaa) [inline Backend/templates/.editorconfig]: 💄 you already have this defined for all the file types, no need to do it here.

**Review by portellaa — APPROVED** [review]
Nice job 🚀

File: Backend/templates/.editorconfig (hunk 2 above)
Comment (tallnato) [reply Backend/templates/.editorconfig]: fixed 🤦‍♂️

**Pull request merged**